asreview/project.py

```diff
@@ -236,7 +304,7 @@ def create(cls,
                 f'Project folder {project_path} already exists.')
 
         try:
-            project_path.mkdir(exist_ok=True)
+            project_path.mkdir(parents=True, exist_ok=True)
```

Comment:
Remove `parents=True`

Reply:
I like having parents=True in there. Pro: it ensures missing parents are created, it enables the end-user to come up with a path that does not exist (less hassle). Con: typos by end-user will be ignored and create a 'wrong' path. Anyways, let me know if you really want to remove this.